newton/solvers/mujoco/solver_mujoco.py

```
@@ -1242,6 +1242,33 @@
                 ],
                 device=model.device,
             )
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)